charts/prow/generate_auth_k8s.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
```

Comment:
Set `errexit`, `nounset`, and `pipefail` at the top of all bash scripts. This means you have to touch some of the logic below, but it shouldn't be too much.

You can get the `exists=$(echo $?)` type of logic by putting it in an `if` statement.